Exiled.API/Enums/SpawnReason.cs

```diff
@@ -10,7 +10,7 @@ namespace Exiled.API.Enums
     /// <summary>
     /// Possible spawn reasons.
     /// </summary>
-    public enum SpawnReason : byte
+    public enum SpawnReason : byte // TOTO: Remove this file and use Basegame
```

Comment:
Why remove this and use basegame when basegame can change it and break plugins, while we can just update the values.

Reply:
i was thinking they was useless to make since NW Have it they just make an other same enum  (for example RoleType we use the BaseGame one not an enum on exiled and so they don't need to be updated)